plugin/evm/vm_test.go

```diff
@@ -2076,86 +2076,96 @@ func TestBuildSubnetEVMBlock(t *testing.T) {
 }
 
 func testBuildSubnetEVMBlock(t *testing.T, scheme string) {
```

Comment:
This is not the intention of this test. `TestBuildSubnetEVMBloc`k ensures it can build block starting with SubnetEVM upgrade see here https://github.com/ava-labs/subnet-evm/blob/JonathanOppenheimer/sync-pr-1127-acp-226-blockgascost/params/extras/network_upgrades.go#L30.

This test was really an ancient one to test-out the initial fork from Coreth. I don't think this is a viable test now, so we can remove this.

the actual fix for this test is to use:
```
ap6Fork := upgradetest.ApricotPhase6
tvm := newVM(t, testVMConfig{
    fork: &ap6Fork
		genesisJSON: genesisJSONSubnetEVM,
		configJSON:  getConfig(scheme, ""),
	})
```

Reply:
removed